examples/src/app/layout.tsx

```diff
@@ -12,7 +13,8 @@ export default function RootLayout({ children }: { children: React.ReactNode })
     <html lang="en">
       <body className="bg-[radial-gradient(at_bottom_left,_var(--tw-gradient-stops))] from-pink-50 to-pink-900">
         <Sidebar>
-          <div className="absolute w-full h-full top-0 pt-14 lg:relative lg:w-auto lg:pt-0 bg-[radial-gradient(at_bottom_left,_var(--tw-gradient-stops))] from-white/90 via-pink-50/90 to-pink-100/90 sm:mr-2 sm:rounded-lg">
+          <CopyLink />
+          <div className="overflow-auto absolute w-full h-full top-0 pt-14 lg:relative lg:w-auto lg:pt-0 bg-[radial-gradient(at_bottom_left,_var(--tw-gradient-stops))] from-white/90 via-pink-50/90 to-pink-100/90 sm:mr-2 sm:rounded-lg">
```

Comment:
This ends up appearing in the table of contents, too, so maybe we hide it from that page?

<img width="1119" alt="Screenshot 2023-11-06 at 4 39 16 PM" src="https://github.com/drifting-in-space/y-sweet/assets/2099249/d42f99d3-be4f-4f1d-b5ac-95d4bb6f24ca">